changelogs/v208-kajabi.1/ruby-bin-path.md

```diff
@@ -0,0 +1,3 @@
+## Support overriding where the Ruby binaries are linked
+
+If a user specifies `ENV["HEROKU_RUBY_RUBY_BIN_DIR"]`, the Ruby binaries (e.g. `ruby`, `gem`, etc.) will be linked to that directorying instead of `./bin`. This allows, for example, having more control of binaries via environment variables or forcing commands to use `bundle` (since `ruby` won't be in the `PATH`).
```

Comment:
```suggestion
If a user specifies `ENV["HEROKU_RUBY_RUBY_BIN_DIR"]`, the Ruby binaries (e.g. `ruby`, `gem`, etc.) will be linked to that directory instead of `./bin`. This allows, for example, having more control of binaries via environment variables or forcing commands to use `bundle` (since `ruby` won't be in the `PATH`).
```